velox/functions/sparksql/SplitFunctions.cpp

```diff
@@ -134,10 +143,148 @@ std::vector<std::shared_ptr<exec::FunctionSignature>> signatures() {
               .build()};
 }
 
+// str_to_map(expr [, pairDelim [, keyValueDelim] ] )
+// Currently only supports single-character pairDelim & keyValueDelim.
```

Comment:
Hi @mbasmanova, I am following up this work.
I just noted that Spark accepts multi-character as delimiter for this function and the delimiter can also be regex pattern. But single-character delimiters are more common in the usage. Can we firstly propose this basic single-character impl.?

Reply:
@PHILO-HE Sure.